packages/dds/task-manager/src/oldestClientObserver.ts

```diff
@@ -53,17 +63,18 @@ import { IOldestClientObserver } from "./interfaces";
  * });
  * ```
  */
-export class OldestClientObserver extends EventEmitter implements IOldestClientObserver {
+export class OldestClientObserver extends TypedEventEmitter<IOldestClientObservableEvents>
```

Comment:
I do not see this class raising connected or disconnected events, so sounds like it does not really implements IOldestClientObservableEvents

Reply:
You're absolutely right, this was a typo and should be `IOldestClientObserverEvents` -- will fix.